architecture/design/ysql-gin-indexes.md

```diff
@@ -0,0 +1,107 @@
+# YSQL GIN indexes
+
+As of v2.6/v2.7, Yugabyte supports only one type of index access method for DocDB-backed relations: `lsm`.  This simply maps columns to the primary key of the base table.
+
+Generalized inverted indexes map elements inside container columns to the pk of the base table.  To support GIN, we add a new access method `ybgin` and implement the access method api.  A lot of the work can be borrowed from upstream postgres's `gin` access method.
```

Comment:
"we" should not be used in tech documentation.
Suggestion:
To support GIN, a new access method `ybgin` has been  added and the access method API implemented.

Reply:
Except that this is instructions for implementation, and not documentation... We is OK here, as it's Yugabyte talking to Yugabyte :)